### ISSUE TITLE
changing learning rate based on epochs.

### ISSUE THREAD

**Issue by pratibhashinde** [fizyr/keras-retinanet#1559]
I saw, there is lr=0.00001. train.py uses ReduceLROnPlateau to change learning rate if validation loss is not decreasing. I am not doing validation while training is going on. How to change this learning rate based on training loss?

Also, I want  to change this learning rate after every 10 epochs. How to do that?
